challenge/src/components/announce/AnnouncementComponent.module.css

```diff
@@ -0,0 +1,9 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 800px;
```

Comment:
gitving height in pixes can get wonky on other screens, for example on my laptop this component is to tall so their are now 2 scroll bars. perhaps using % instead may help, or doing `@media` screen queries if you want to be extra fancy
![Screenshot from 2020-06-09 23-08-11](https://user-images.githubusercontent.com/314083/84222696-3513bb00-aaa6-11ea-8e5d-e06f319ebe71.png)

Reply:
vh seems to do the trick!